### DISCUSSION TITLE
Is it possible to set the tab I want when open the GUI window?

### DISCUSSION THREAD

**Discussion by d1658999** [alejandroautalan/pygubu-designer#180] [Q&A] [answered]
As title

I have numbers of tabs in mainwindow

and I want a specific tab is fixed when opening GUI every time like temp PSU not FTM

how to achieve this action?

![capture](https://user-images.githubusercontent.com/43318985/206857319-1a27fa44-eb09-4552-bb3c-c14f784ed7dc.PNG)

thanks

BRs
Jack

**Comment by alejandroautalan** [comment, marked as answer]
Hello @d1658999 thanks for trying pygubu.

> I have numbers of tabs in mainwindow and I want a specific tab is fixed when opening GUI every time like temp PSU not FTM how to achieve this action?

You can do this by code.  I uploaded an example here:

https://github.com/alejandroautalan/pygubu-designer/blob/6a05d4205615eeb70334f8691dc0ecfbbdee71f2/examples/notebook/demo1app.py#L25-L29

Regards
Alejandro A.

**Comment by d1658999** [comment]
Thanks for good and useful reply!

I also provide a good useful information for someone who want to learn this GUI layout tool:

it also can use _index_ to return the index of current tab

`tab_index = self.notebook.index(self.notebook.select())`

BRs
Jack
